dist/backdraft.js

```diff
@@ -532,6 +532,16 @@ _.extend(Plugin.factory, {
       return !columnConfig.bulk;
     });
 
+    // read initial filters from URL
+    var urlParamString = window.location.href.split("?")[1];
+    if (urlParamString && $.deparam(urlParamString) && $.deparam(urlParamString).ext_filter_json ) {
```

Comment:
I agree we discussed renaming ext_filter_json but also I think we talked about still accepting ext_filter_json to make sure we support bookmarked links.